api/database/migrations/2020_08_27_104237_add_location_to_events_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddLocationToEventsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('events', function (Blueprint $table) {
+            $table->string('location')->nullable();
```

Comment:
Probably we will need one more field - `location_type` (ex. address, zoom etc.). `location` can be used as the data for `location_type` (ex. place name or coordinates for address, URL for zoom etc.)

Reply:
do I need to add a location type field?

Reply:
ok, you can skip it for now